src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,107 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private int size;
+    private int capacity;
+    private float loadFactor;
```

Comment:
removing capacity and loadFactor fields will make the code less bulky and more readable. Moreover, loadFactor shouldn't even change, you have a constant for that